modules/op-signing-secrets-in-tekton-chains.adoc

```diff
@@ -39,6 +39,54 @@ Cosign prompts you for a password, and creates a Kubernetes secret.
 
```

Comment:
@vdemeester cosign is not productized, are we supporting it for GA? Epic https://issues.redhat.com/browse/SRVKP-2199 only talks about `skopeo`

Reply:
I don't think we "support" cosign yet, but I think we can still document things with it ? 🤔 
Also it's being used and advertised by rhtap, … so my guess is that red hat will relatively soon-ish support it ?

Reply:
This is an existing part of the dic, I would probably want to go through program management if I outright remove it

Reply:
@vdemeester @ppitonak do you want me to try and go to Program Management about whether we want to remove cosign doc or can we let this slide?

Reply:
@vdemeester I did now send a question to Koustav, but I see an additional problem here. There are two more places where cosign commands are used:

https://59627--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/using-tekton-chains-for-openshift-pipelines-supply-chain-security.html#using-tekton-chains-to-sign-and-verify-image-and-provenance_using-tekton-chains-for-openshift-pipelines-supply-chain-security specifies cosign as a prerequisite and uses it to generate a key. I guess I can replace with Skopeo using the commands already documented. But then it also uses `cosign verify` and `cosign verify-attestation`, how do I do this with skopeo?

https://59627--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/using-tekton-chains-for-openshift-pipelines-supply-chain-security.html#creating-and-verifying-task-run-signatures-without-any-additional-authentication_using-tekton-chains-for-openshift-pipelines-supply-chain-security uses cosign to verify the signature in step 7, how do I do this with skopeo?

Reply:
@mramendi 
We should keep both Skopeo and Cosign for now. Skopeo has only shipped functionality for generating sigstore keys, not for verifying the signatures yet which is expected to come in future. So for generating keys, we should mention both skopeo and cosign. For verifying signatures, we should keep the documentation with cosign. Later, once Skopeo has all the required functionality for using sigstore keys in Chains shipped, we can take a decision to either keep/remove cosign, the decision is also dependent on how customers are consuming whether via skopeo or cosign. Hope this makes sense. 

cc: @vdemeester @ppitonak

Reply:
Thanks, resolved.